src/api/operations/customCaseNoteQueries.graphql

```diff
@@ -49,6 +49,7 @@ query GetClientCaseNotes(
           projectType
           entryDate
           exitDate
+          autoExited
```

Comment:
`autoExited` isn't used by custom case notes, but the custom case notes return type is passed to `EnrollmentDateRangeWithStatus` [here](https://github.com/greenriver/hmis-frontend/blob/c37b83c10d89e6bf897e65d8241ad8c0a3eb1c44/src/modules/caseNotes/ClientCaseNotes.tsx#L34).

I thought maybe it's neater to add this here rather than use a partial, since it's just a boolean - but now that I'm thinking about it, maybe it's not good because on the backend we need to join with the Exit table in order to fetch this information.  @gigxz what do you think?

Reply:
It won't cause an extra query because we already join with `exit` to get the exit date here.

But, I think it would make more sense to resolve the enrollment using the  `ClientEnrollmentFields` fragment, since that's what we do everywhere else when we resolve records on client (`GetClientFiles`, `GetClientServices`, `GetClientAssessments`, etc.). I don't think there's any reason Case Notes are different, they probably just got missed when we tightened that up. It will resolve additional fields that we don't _necessarily_ need currently, but I think it's worth it for the consistency.

Reply:
Ah okay, yeah this makes sense. Thanks for the neater suggestion!